uktrade/enquiry-mgmt-tool
# Add UK as a region option

## Description of change

This PR adds `UK` to the list of Region choices. This will allow users to select the UK as an option in the Region dropdown in the edit view. 

## Test instructions

- Go to an enquiry
- Click edit
- Go to the region field and select UK
- Check the UK is saved as the region for enquiry
